libretroshare/src/retroshare/rsdisc.h

```diff
@@ -30,20 +30,55 @@
 
 /* The Main Interface Class - for information about your Peers */
 class RsDisc;
+
+/**
+ * Pointer to global instance of RsDisc service implementation
+ * @jsonapi{development}
+ */
 extern RsDisc   *rsDisc;
 
 class RsDisc
 {
 	public:
 
 	RsDisc()  { return; }
-virtual ~RsDisc() { return; }
+	virtual ~RsDisc() { return; }
```

Comment:
now that you edited the line please take the chance to remove the horrible `return;` on the empty method